include/networkit/generators/BarabasiAlbertGenerator.hpp

```diff
@@ -18,57 +18,67 @@ namespace NetworKit {
+     * Albert[1], implemented in the much faster method from Batagelj and Brandes[2] per default
+     * where the running time is O(n+m). Furthermore there is a parallel version from Sanders and
+     * Schulz[3] implemented which can be selected by setting sequential=false which empirically
+     * showed better runtimes (when executed multi-threaded).
+     *
      * [1] Barabasi, Albert: Emergence of Scaling in Random Networks
      *   http://arxiv.org/pdf/cond-mat/9910332.pdf
      * [2] ALG 5 of Batagelj, Brandes: Efficient Generation of Large Random Networks
      *   https://kops.uni-konstanz.de/bitstream/handle/123456789/5799/random.pdf?sequence=1
-     * Running time is O(n+m)
+     * [3] Peter Sanders, Christian Schulz: Scalable generation of scale-free graphs
+     *   https://www.sciencedirect.com/science/article/pii/S0020019016300102
      *
      * The generator will emit a simple graph, where all
      * new nodes are initially connected to k random neighbors.
      *
      * @param k     Number of attachments per node
      * @param nMax  Number of nodes in the graph
      * @param n0    Number of connected nodes to begin with
-     * @param batagelj Specifies whether to use Batagelj and Brandes's method (much faster)
-     *              rather than the naive one; default: true
+     * @param sequential Specifies whether to use Batagelj and Brandes's method (sequential) or a
+     * parallel variant. Default: true.
      */
-    BarabasiAlbertGenerator(count k, count nMax, count n0 = 0, bool batagelj = true);
+    BarabasiAlbertGenerator(count k, count nMax, count n0 = 0, bool sequential = true);
 
     /**
-     * This generator implements the preferential attachment model as introduced by Barabasi and
-     * Albert[1]. The original algorithm is very slow and thus, the much faster method from Batagelj
-     * and Brandes[2] is implemented and the current default. The original method is no longer
-     * supported and the \p batagelj parameter will be removed in future releases.
+     * This generator uses the preferential attachment model as introduced by Barabasi and
+     * Albert[1], implemented in the much faster method from Batagelj and Brandes[2] per default
+     * where the running time is O(n+m). Furthermore there is a parallel version from Sanders and
+     * Schulz[3] implemented which can be selected by setting sequential=false which empirically
+     * showed better runtimes (when executed multi-threaded).
+     *
      * [1] Barabasi, Albert: Emergence of Scaling in Random Networks
      *   http://arxiv.org/pdf/cond-mat/9910332.pdf
      * [2] ALG 5 of Batagelj, Brandes: Efficient Generation of Large Random Networks
      *   https://kops.uni-konstanz.de/bitstream/handle/123456789/5799/random.pdf?sequence=1
-     * Running time is O(n+m)
+     * [3] Peter Sanders, Christian Schulz: Scalable generation of scale-free graphs
```

Comment:
As a follow-up from the review of the Chung-Lu Model PR. We should be consistent with how we cite references in the comments. For URLs, use markdown style. Using reference numbers certainly makes the comments more readable. I suggest transferring the style from this PR also to the Chung-Lu PR.